### ISSUE TITLE
Constant segfaults on Ubuntu 16.04

### ISSUE THREAD

**Issue by mjeffe** [OpenShot/openshot-qt#392]
I had high hopes for the 2.0 version, but it segfaults so often, it's unusable. My hardware is nothing out of the ordinary a one year old Dell Latitude E5550, 16GB RAM, integrated video.

When run from the command line I usually see some kind of error just as is dumps core. Here is an example:

```
preview_thread:INFO self.player.Position(): 85
 main_window:INFO main::addSelection: item_id: , item_type: effect, clear_existing: True
 main_window:INFO main::addSelection: item_id: 2LK2LPK9SX, item_type: clip, clear_existing: True
[ac3 @ 0x7f94cc484160] frame sync error
[AudioData @ 0x7f947b7fd700] invalid NULL pointer for src[0]
Segmentation fault (core dumped)
```

I am more than happy to help run other tests if it will help you improve the software. I appreciate and applaud your open source contribution. Obviously lots of people have found it useful, unfortunately every time I have tried it, I have walked away disappointed.

**Comment by chibill** [comment]
I was plagued with the same problem. Gave up on openshots but as even the rendered video is chopy (flickers every now and then. ) And took an hour to render 10 min...

**Comment by nigels-com** [comment]
Seeing instability on Ubuntu 16.04 here too, via PPA.

**Comment by nigels-com** [comment]
The AppImage won't run on Ubuntu 16.04, apparently due to a Qt version mismatch.

**Comment by ankitpati** [comment]
@nigels-com The 2.2 AppImage runs correctly in my testing.

Can everyone here please verify if the issue continues? If not, we can mark it as fixed.

**Comment by mjeffe** [comment]
Hey, this is loads better!  I haven't had a lot of time to mess with it in the last couple of days, but what I have done gives me hope. I was able to run through an entire sequence of creating a project, importing a couple of clips, cutting, splicing, adding transitions, and exporting. The only "crash" happened when, in the "Export Video" dialog, I  1) I clicked the "Target: " video codec field, 2) scrolled through the options and 3) clicked cancel. At this point, Ubunut's apport-gtk pops up and said "openshot-qt has crashed, relaunch or leave closed". I was able to consistently recreate the behavior, however only once did openshot actually freeze up. Every other time openshot continued to work even though apport-gtk reported a crash.

I'm excited about this new build. Openshot actually feels usable.

BTW, I'm still on the same hardware and OS as in my comment above. I used the ppa to install as described on the web site http://www.openshot.org/ppa/
